litestar/middleware/exceptions/middleware.py

```diff
@@ -37,6 +32,8 @@
     )
     from litestar.types.asgi_types import WebSocketCloseEvent
 
+__all__ = ("ExceptionHandlerMiddleware", "ExceptionResponseContent", "create_exception_response")
+
 
 def get_exception_handler(exception_handlers: ExceptionHandlersMap, exc: Exception) -> ExceptionHandler | None:
     """Given a dictionary that maps exceptions and status codes to handler functions, and an exception, returns the
```

Comment:
do we add a `.. versionchanged::` directive?

Reply:
I don't _think_ so. AFICT, exception handler middleware is undocumented, and this shouldn't change the behavior of the application at all.